helm-chart/kuberay-apiserver/values.yaml

```diff
@@ -11,6 +11,9 @@ image:
   tag: nightly
   pullPolicy: IfNotPresent
 
+cors:
+  # allowOrigin: "*"
```

Comment:
What does "*" mean? Can you add some comments to explain what are valid values and the expected behavior?